onnx/reference/ops/_helpers.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: Apache-2.0
+from typing import Any, Dict, Union
+
+from onnx.reference.op_run import OpRun, _split_class_name
+
+
+def build_registered_operators_any_domain(
+    module_context: Dict[str, Any]
+) -> Dict[str, Dict[Union[int, None], OpRun]]:
+
+    reg_ops: Dict[str, Dict[Union[int, None], OpRun]] = {}
+    for class_name, class_type in module_context.items():
+        if class_name.startswith("_") or class_name in {
+            "Any",
+            "Dict",
+            "List",
+            "TOptional",
+            "Union",
+            "cl",
+            "class_name",
+            "get_schema",
+            "module_context",
+            "textwrap",
+        }:
+            continue
+        if isinstance(class_type, type(build_registered_operators_any_domain)):
+            continue
+        try:
+            issub = issubclass(class_type, OpRun)
+        except TypeError as e:
+            raise TypeError(
+                f"Unexpected variable type {class_type!r} and class_name={class_name!r}."
+            ) from e
+        if issub:
+            op_type, op_version = _split_class_name(class_name)
+            if op_type not in reg_ops:
+                reg_ops[op_type] = {}
+            reg_ops[op_type][op_version] = class_type
+    if not reg_ops:
+        raise RuntimeError(
+            f"No registered operator. This error happens when no implementation "
+            f"of type OpRun was detected. It is suggested to reinstall the package."
```

Comment:
```suggestion
            f"of type 'OpRun' was detected. It may be due to an error during installation. Please try reinstalling onnx."
```

nit: clarify it is an onnx error, in case it is raised in a package consuming onnx?